samples/native/test/main.c

```diff
@@ -23,37 +23,36 @@
 
 /* Private Data. */
 /* All static data definitions appear here. */
-RequestServiceInfo rsi = { 
-  .ExecutionType  = "native",
-  .ExeCmd         = "ls"
+RequestServiceInfo rsi = {
+	.ExecutionType  = "native",
```

Comment:
@tdrozdovsky Question. Is this the 4 spaces that we suggested? Or did you use the "gofmt" plugin for the right indentation? Just for another self clarification.